client/components/reader-infinite-stream/index.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * External Dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import {
+	List,
+	WindowScroller,
+	CellMeasurerCache,
+	CellMeasurer,
+	InfiniteLoader,
+	AutoSizer,
+} from 'react-virtualized';
+import { debounce, noop, get } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import ConnectedSubscriptionListItem from 'blocks/reader-subscription-list-item/connected';
+
+class ReaderInfiniteStream extends Component {
+	static propTypes = {
+		items: PropTypes.array.isRequired,
+		fetchNextPage: PropTypes.func,
+		forceRefresh: PropTypes.any, // forceRefresh can be anything. Whenever we want to force a refresh, it should change
+		windowScrollerRef: PropTypes.func,
+		showLastUpdatedDate: PropTypes.bool,
+		followSource: PropTypes.string,
+		itemType: PropTypes.oneOf( [ 'site', 'post' ] ),
+	};
+
+	static defaultProps = {
+		windowScrollerRef: noop,
+		showLastUpdatedDate: true,
+		itemType: 'site',
+	};
+
+	heightCache = new CellMeasurerCache( {
+		fixedWidth: true,
+		minHeight: 70,
+	} );
+
+	// eslint-disable-next-line no-unused-vars
+	postRowRenderer = ( { index, key, style, parent } ) => {
+		// todo: implement
+		return null;
+	};
+
+	// TODO: why doesn't this work?
+	siteRowRenderer = ( { index, key, style, parent } ) => {
+		const { followSource, showLastUpdatedDate, items } = this.props;
+		const site = items[ index ];
+
+		const feedUrl = get( site, 'feed_URL' );
```

Comment:
siteRowRenderer didn't work but oldSiteRowRenderer did.  `siteRowRenderer` is the more composable version because of how it builds off of `measuredRowRenderer`.  I fixed this particular issue in its separate branch